packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts

```diff
@@ -128,7 +180,82 @@ describe('TransferDomain', () => {
       await expect(promise).rejects.toThrow('Src address must be a legacy or Bech32 address in case of "DVM" domain')
     })
 
-    it('(evm -> dvm) should fail if source address and source domain are not match', async () => {
+    it('(evm -> dvm) should fail if dst address is not legacy or Bech32 address in case of "DVM domain', async () => {
+      const addressP2SH = await container.getNewAddress('address', 'p2sh-segwit')
+      // initially transfer DFI
+      await client.account.transferDomain([
+        {
+          src: {
+            address: dvmAddr, // invalid
```

Comment:
```suggestion
            address: dvmAddr,
```

Reply:
fixed in commit: https://github.com/BirthdayResearch/jellyfishsdk/pull/2138/commits/a98ef131b35f43d35281f97702b491abff86bc18

Reply:
resolved in https://github.com/BirthdayResearch/jellyfishsdk/pull/2138/commits/abd161b451be096bc1c7018bb462b18b37fa2615